stable/mongodb/values.yaml

```diff
@@ -307,11 +307,15 @@ metrics:
     initialDelaySeconds: 15
     periodSeconds: 5
     timeoutSeconds: 5
+    failureThreshold: 3
+    successThreshold: 1
   readinessProbe:
     enabled: false
     initialDelaySeconds: 5
     periodSeconds: 5
     timeoutSeconds: 1
+    failureThreshold: 3
+    successThreshold: 1
```

Comment:
Please ensure the same changes are applied in the **values-production.yaml** file too.

Reply:
added the same changes in **values-production.yaml** :heavy_check_mark: